conf/ssh/sshd_config

```diff
@@ -103,7 +99,7 @@ Match Group sftp.app,!ssh.app
     AllowStreamLocalForwarding no
     PermitTunnel no
     PermitUserRC no
-    PasswordAuthentication yes
+    PasswordAuthentication {{ password_authentication }}
```

Comment:
Dunno what to think about this, the point of it being `yes` is for the use-case of, for example, `my_webapp`, which you may want to access via SFTP and only a password is provided ... (+ dunno how convenient/easy it is to configure Filezilla to use an SSH key rather than a password ...)

Reply:
Right, but if "consistency" equals "nobody understands how to use some features, such as connecting to my_webapp using SFTP because it's not properly documented or is overly complex" then it's still an issue

It's not just about configuring FileZilla, it's also about adding the SSH key to the `my_webapp` user, which is not a trivial operation for not-so-tech-savvy users

Reply:
Then, this people will stick with password authentication anyway, or learn how to do with an SSH key if they really want. Most of cloud providers allow key authentication. It's even sometimes the only authentication method
to avoid having (encrypted/hashed) password on the cloud side.

Reply:
Another alternative will be to have something similar to webadmin_allowlist for SSH to defined trusted IPs with PermitRootLogin and PasswordAuthentication. This will solved the point of #1551.